File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/test/ConstantsSaverTest.java
package org.firstinspires.ftc.teamcode.opmodes.test;

import com.arcrobotics.ftclib.command.CommandOpMode;
import com.qualcomm.robotcore.eventloop.opmode.TeleOp;

import org.firstinspires.ftc.teamcode.constants.ConstantsSaver;

@TeleOp(name = "Test - Constants Saver")
public final class ConstantsSaverTest extends CommandOpMode {

    @Override public void initialize() {
        new ConstantsSaver(telemetry).save();
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/test/ConstantsSaverTest.kt
package org.firstinspires.ftc.teamcode.opmodes.test

import com.arcrobotics.ftclib.command.CommandOpMode
import com.qualcomm.robotcore.eventloop.opmode.TeleOp
import org.firstinspires.ftc.teamcode.constants.ConstantsSaver

@TeleOp(name = "Test - Constants Saver", group = "Test")
class ConstantsSaverTest: CommandOpMode() {

    override fun initialize() {
        ConstantsSaver(telemetry).save()
    }
}